### PR TITLE
RAD-108: Correcting Guidewindow Start Stop

### PR DESCRIPTION
**Pull request by PaulHuwe**
Ticket to change the variable type of x/y start/stop/size in guidewindow schema. Added those keywords to the guidestar schema as well.

### CODE REVIEW

**Review by ddavis-stsci — APPROVED** [review]
LGTM

**Review by schlafly — APPROVED** [review]

File: src/rad/resources/schemas/guidestar-1.0.0.yaml [hunk 1]
```diff
@@ -198,11 +198,33 @@ properties:
     archive_catalog:
       datatype: float
       destination: [ScienceCommon.gs_pattern_error]
+  gw_window_xstart:
```
Comment (WilliamJamieson) [inline src/rad/resources/schemas/guidestar-1.0.0.yaml:201]: Do these also need archive metadata?

**Review by kdupriestsci — COMMENTED** [review]
Is it normal to have the same attribute in multiple files?  I see gw_window* (title and type) added to guidestar.yaml, but they still exist in guidewindow.yaml with title, type, and others defined.

**Review by kdupriestsci — APPROVED** [review]

File: src/rad/resources/schemas/guidestar-1.0.0.yaml (hunk 1 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/guidestar-1.0.0.yaml:201]: Nope, not in guidestar.

**Pull request merged**